ktext.md

```diff
@@ -82,27 +82,42 @@ Without the below 2, no system is bootable:
   * A kext to patch many processes, required for AppleALC, WhateverGreen, VirtualSMC and many other kexts. Without Lilu, they will not work.
   * Note that while Lilu supports as early as Mac OS X 10.4, many plugins only work on newer versions.
 
+:::
+
 ### VirtualSMC Plugins
 
-The below plugins are not required to boot, and merely add extra functionality to the system like hardware monitoring (Note while VirtualSMC supports 10.4, plugins may require newer versions):
+The below plugins are not required to boot, and merely add extra functionality to the system like hardware monitoring. Unless otherwise specified, these plugins come with VirtualSMC
+
+::: tip
+
+While VirtualSMC supports 10.4, plugins may require newer versions.
+
+:::
 
 * SMCProcessor.kext
-  * Used for monitoring CPU temperature, **doesn't work on AMD CPU based systems**
+  * Used for monitoring Intel CPU temperature
+  * Not for AMD CPU based systems
   * Requires Mac OS X 10.7 or newer
 * [SMCAMDProcessor](https://github.com/trulyspinach/SMCAMDProcessor)
-  * USed for monitoring CPU temperature on AMD Zen-based systems
+  * Used for monitoring CPU temperature on AMD Zen-based systems
   * **Under active development, potentially unstable**
-  * Requires AMDCPURyzenPowerManagement (see [AMD CPU Specific Kexts](ktext.md#amd-cpu-specific-kexts))
+  * Requires AMDRyzenCPUPowerManagement (see [AMD CPU Specific Kexts](ktext.md#amd-cpu-specific-kexts))
   * Requires macOS 10.13 or newer
+* [SMCRadeonGPU](https://github.com/aluveitie/RadeonSensor)
+  * Used for monitoring GPU temperature on AMD GPU systems
+  * Requires RadeonSensor from the same repository
+  * Requires macOS 11 or newer
 * SMCSuperIO.kext
-  * Used for monitoring fan speed, **doesn't work on AMD CPU based systems**
+  * Used for monitoring fan speed
+  * Not for AMD CPU based systems
   * Requires Mac OS X 10.6 or newer
 * SMCLightSensor.kext
-  * Used for the ambient light sensor on laptops, **desktops can ignore**
-  * Do not use if you don't have an ambient light sensor, can cause issues otherwise
+  * Used for the ambient light sensor on laptops
+  * **Do not use if you don't have an ambient light sensor (ie. desktops), can cause issues otherwise**
```

Comment:
Can add a fake light sensor to add screen dimming on battery. Probably not relevant in the install guide though.

Reply:
Agreed